bin/live/pycbc_live_combine_single_fits

```diff
@@ -0,0 +1,198 @@
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+
+import h5py, numpy as np, argparse
+import matplotlib
+matplotlib.use('agg')
+from matplotlib import pyplot as plt
+import re, datetime
+
+parser = argparse.ArgumentParser(usage="",
+    description="Plot histograms of triggers split over various parameters")
+parser.add_argument("--verbose", action="store_true",
+                    help="Print extra debugging information", default=False)
+parser.add_argument("--trfits-files", nargs="+", required=True,
+                    help="Files containing daily trigger fits")
+parser.add_argument("--output-file", required=True,
+                    help="Output hdf file for final results")
+parser.add_argument("--output-plot-file-format",
+                    help="Output file format for alphas vs time plots, needs to "
+                         "contain '{}' as marker for where ifo string is included")
+parser.add_argument("--ifos", required=True, nargs="+",
+                    help="list of ifos fo collect info for")
+
+args=parser.parse_args()
+
+colours = ['r',(1.0,0.6,0),'y','g','c','b','m','k',(0.8,0.25,0),(0.25,0.8,0)]
+
+counts_all = {ifo:[] for ifo in args.ifos}
+alphas_all = {ifo:[] for ifo in args.ifos}
+analysis_dates = []
+
+files = args.trfits_files
+
+fit_f0 = h5py.File(files[0], 'r')
+
+bl = fit_f0['bins_lower'][:]
+bu = fit_f0['bins_upper'][:]
+
+live_times = {ifo : [] for ifo in args.ifos}
+
+for f in files:
+    fits_f = h5py.File(f,'r')
+    if 'days_since_epoch' in fits_f.attrs:
+        analysis_dates += [fits_f.attrs['days_since_epoch']]
+    elif 'analysis_date' in fits_f.attrs:
+        analysis_year = int(fits_f.attrs['analysis_date'][:4])
+        analysis_month = int(fits_f.attrs['analysis_date'][5:7])
+        analysis_day = int(fits_f.attrs['analysis_date'][8:10])
+        n_since_epoch = (datetime.date(analysis_year, analysis_month, analysis_day)\
+                         - datetime.date(2000,1,1)).days 
+        analysis_dates += [n_since_epoch]
+    else:
+        re_string = '([12]\d{3}_(0[1-9]|1[0-2])_(0[1-9]|[12]\d|3[01]))'
+        m = re.search(re_string, f)
+        analysis_year = int(m.group(0)[:4])
+        analysis_month = int(m.group(0)[5:7])
+        analysis_day = int(m.group(0)[8:10])
+        n_since_epoch = (datetime.date(analysis_year, analysis_month, analysis_day)\
+                         - datetime.date(2000,1,1)).days
+        analysis_dates += [n_since_epoch]
+    for ifo in args.ifos:
+        if ifo not in fits_f:
+            counts_all[ifo] += [-1 * np.ones_like(counts_all[ifo][-1])]
+            alphas_all[ifo] += [-1 * np.ones_like(alphas_all[ifo][-1])]
+            print(f + " has no " + ifo + " triggers")
```

Comment:
Seems like you're using a mix of bare `print` statements and logging. Personally, I'd pick one or the other, and I'd use logging.